src/transports/smart_pkt.c

```diff
@@ -309,8 +311,8 @@ static int ng_pkt(git_pkt **out, const char *line, size_t len)
 	if (len < 1)
```

Comment:
This line doesn't make a lot of sense in this context then, either, does it?

```
line = ptr + 1;
if (line >= eol)
    goto out_err;
```

Would make a lot more sense.

Reply:
Thank you @pks-t , that was missing! I added it to my previous commit.